SixTrack/sixtrack.s

```diff
@@ -16782,8 +16782,9 @@ cc2008
      &'vert. tilt         = ',wire_tilty(j),' degrees'
 ! ignore wire if current, length or displacment are 0 or
 ! wire_flagco not set (case wire_flagco = 0)
+! for displacement only ignore if wire_dispx = wire_dispy = 0
             if( abs(wire_flagco(j)*wire_current(j)*wire_lint(j)
-     &*wire_lphys(j)*wire_dispx(j)*wire_dispy(j)).le.pieni ) then
+     &*wire_lphys(j)*(wire_dispx(j)+wire_dispy(j))).le.pieni ) then
```

Comment:
Could you please paranthesize this a*b*c*d... We don't want some binaries switching it into a marker and some not.